models/Dynamic_Compression_Model/hdlsrc/sm_DynamicCompression/DynamicCompressionWithRx.c

```diff
@@ -331,9 +331,9 @@ static int DynamicCompressionWithRx_remove(struct platform_device *pdev) {
 Generated by CreateExitFunction
 ************************************************/
 static void DynamicCompressionWithRx_exit(void) {
-  pr_info("Flat Earth DynamicCompressionWithRx module exit\n");
+  pr_info("Audio Logic
   platform_driver_unregister(&DynamicCompressionWithRx_platform);
-  pr_info("Flat Earth DynamicCompressionWithRx module successfully unregistered\n");
```

Comment:
This breaks the C file since it will no longer successfully compile. This is also true of all the rest of the generated C files in the repo.